Feature/compose swift bridge only classes

JuliaJakubcova opened the pull request:
# :pencil: Description
- Add classes compose-swift-bridge plugin would add, but without the plugin (because it's buggy)

# :bulb: What’s new?
- PlatformSpecificCheckbox with SwiftUI implementation and necessary factories 

# :no_mouth: What’s missing?
- Nothing

# :books: References
- FIXME

tomas-bat commented:
Does this contain stuff from #31 ? 🤔

JuliaJakubcova commented:
> Does this contain stuff from #31 ? 🤔

It did at first, but then I rewrote it and added all the classes manually, without using the plugin, because it contains bugs (for now) 😄

tomas-bat commented:
> It did at first, but then I rewrote it and added all the classes manually, without using the plugin, because it contains bugs (for now) 😄

So what's gonna happen with #31? Is this supposed to be merged instead?

JuliaJakubcova commented:
> > It did at first, but then I rewrote it and added all the classes manually, without using the plugin, because it contains bugs (for now) 😄
> 
> So what's gonna happen with #31? Is this supposed to be merged instead?

I think that's what @Seidak decided, if I remember correctly, that we would merge this and keep the branch with the plugin usage separate
